### PR TITLE
Prayers page

### PR DESCRIPTION
**Pull request by mrvovanness**

### CODE REVIEW

File: app/views/prayers/index.html.erb [hunk 1]
```diff
@@ -1,31 +0,0 @@
...
-        <td><%= link_to 'Details', prayer %></td>
-        <% if prayer.user == current_user %>
-        <td><%= link_to 'Edit', edit_prayer_path(prayer) %></td>
-        <td><%= link_to 'Delete', prayer, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-            <% end %>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
-
```
Comment (mrvovanness) [inline app/views/prayers/index.html.erb]: n _feedall.html.slim

can we addd 3rd line here?
```
.panel.panel-default
h3 Pray for #{ link_to prayer.user_name, user_path(prayer.user) } for the following... #{ prayer.content }
h6 #{prayer.likers(User).each.count} person has prayed
```

Comment (pyralis) [inline app/views/prayers/index.html.erb]: yes, add something like that -- maybe we also need code to change based on how many users have prayed

File: app/controllers/prayers_controller.rb [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
   before_action :set_prayer, only: [:show, :edit, :update, :destroy]
-  before_filter :authenticate_user!, only: [:create, :new, :edit, :destroy]
+  before_action :authenticate_user!
 
   def index
-    @prayers = Prayer.all
+    @my_prayers = current_user.prayers
+    @whos_praings = current_user.likeables(Prayer)
+    @supports = current_user.supports
   end
```
Comment (mrvovanness) [inline app/controllers/prayers_controller.rb]: actually.... maybe even in controler we do 
```
if prayer.likers(User).each.count = 0
Praynumber= "no one has prayed for this yet"
esleif prayer.likers(User).each.count = 0
praynumber = "One person has prayed for this"
else 
praynumber = "#{prayer.likers(User).each.count} have prayed for this"
```
Sorry I did not test that code it is probably wrong but you'll ahve the idea.

File: app/views/layouts/_header.html.slim [hunk 3]
```diff
@@ -0,0 +1,24 @@
+nav.navbar.navbar-default role='navigation'
+  /Brand and toggle get grouped for better mobile display
+  .container
+  .navbar-header
+    button type='button' class='navbar-toggle' data-toggle='collapse' data-target='.navbar-ex1-collapse'
+      span class="sr-only" Toggle navigation
+      span class="icon-bar"
+      span class="icon-bar"
+      span class="icon-bar"
```
Comment (mrvovanness) [inline app/views/layouts/_header.html.slim]: Can you expain what is doing the above code? Why three times "icon tab" here?

File: app/views/prayers/index.html.erb (hunk 1 above)
Comment (mrvovanness) [inline app/views/prayers/index.html.erb]: I think may be better add counter inside button pray/praying. Much prettier :)

File: app/views/layouts/_header.html.slim (hunk 3 above)
Comment (pyralis) [inline app/views/layouts/_header.html.slim]: Not sure whats going on there? I think we can remove the extra iconbar lines....

Comment (mrvovanness) [inline app/views/layouts/_header.html.slim]: ok I was don't sure about these lines and their meaning :)

**Pull request merged**